src/Settings.php

```diff
@@ -235,6 +235,24 @@ public static function getPathsFromStdIn()
         $lines = explode("\n", rtrim($content));
         return array_map('rtrim', $lines);
     }
+
+    /**
+     * @param string $path
+     * @param string $os
+     * @return string
+     */
+    public static function escapeString(string $path, string $os = PHP_OS)
+    {
+        $path = trim($path);
```

Comment:
now my program can't be in `/home/user/bin `. it's unrealistic, but you remove this option with this code without any good reason.